src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const tHeadTags = document.querySelectorAll('th');
+const tbody = document.querySelector('tbody');
+const rows = tbody.querySelectorAll('tr');
```

Comment:
Verify that `rows` is not empty to ensure that there are rows to sort. This will prevent unnecessary operations if the table is empty.